op-service/dial/rollupclient_interface.go

```diff
@@ -11,10 +11,16 @@ import (
 // RollupClientInterface is an interface for providing a RollupClient
 // It does not describe all of the functions a RollupClient has, only the ones used by the L2 Providers and their callers
 type RollupClientInterface interface {
+	RollupSyncer
 	OutputAtBlock(ctx context.Context, blockNum uint64) (*eth.OutputResponse, error)
-	SyncStatus(ctx context.Context) (*eth.SyncStatus, error)
 	RollupConfig(ctx context.Context) (*rollup.Config, error)
 	StartSequencer(ctx context.Context, unsafeHead common.Hash) error
 	SequencerActive(ctx context.Context) (bool, error)
 	Close()
 }
+
+// RollupSyncer is the interface of a rollup client from which its sync status
+// can be queried.
+type RollupSyncer interface {
```

Comment:
`RollupSyncer` is a slightly weird name. Maybe `RollupSyncStatus` or `SyncStatusProvider` - I get what you're for with `Syncer`

Reply:
True, picked `SyncStatusProvider`